src/core/annotation.js

```diff
@@ -1048,13 +1048,14 @@ class PopupAnnotation extends Annotation {
 
     let parentSubtype = parentItem.get('Subtype');
     this.data.parentType = isName(parentSubtype) ? parentSubtype.name : null;
-    this.data.parentId = dict.getRaw('Parent').toString();
+    const rawParent = dict.getRaw('Parent');
+    this.data.parentId = rawParent ? rawParent.toString() : null;
 
     if (parentItem.has('RT')) {
```

Comment:
This can be made a lot simpler, and the `.has(...)` check is unnecessary here as well, if you look at the [definition of the `isName` helper function](https://github.com/mozilla/pdf.js/blob/d0892e46e2b0d16ecd7f720f36f36202ae1c12cd/src/core/primitives.js#L284-L286):
```javascript
const rt = parentItem.get('RT');
if (isName(rt, AnnotationReplyType.GROUP)) {
  ...
}
```

Reply:
That's cool, thank you :) ... didn't cross my mind that it can have two arguments.